client/reader/following-manage/index.jsx

```diff
@@ -96,10 +92,19 @@ class FollowingManage extends Component {
 			debounce( this.resizeSearchBox, 50 )
 		);
 		this.resizeSearchBox();
+
+		// this is a total hack. In React-Virtualized you need to tell a WindowScroller when the things
+		// above it has moved with a call to updatePosision().  Our issue is we don't have a good moment
+		// where we know that the content above the WindowScroller has settled down and so instead the solution
+		// here is to call updatePosition in a regular interval. the call takes about 0.1ms from empirical testing.
+		this.updatePosition = setInterval( () => {
+			this.windowScrollerRef && this.windowScrollerRef.updatePosition();
+		}, 100 );
```

Comment:
i mean...it can be lowered. I figured at 100ms the issue would be imperceptible.  And seeing as the operation takes 0.1ms then I believe this interval to be okay perf wise.

I'm less concerned about the actual number (but i'd be happy to change it) and more with the overall approach. 🤷‍♂️